paperspace/workspace.py

```diff
@@ -24,11 +24,14 @@ def __init__(self, experiments_api, logger=None):
         self.logger = logger or default_logger
 
     @staticmethod
-    def _retrieve_file_paths(dir_name, ignored_files=''):
+    def _retrieve_file_paths(dir_name, ignored_files=None):
```

Comment:
This is wrong - if you will keep None you then have to do a check if its not None - this is regression and not a python way - also what is the benefit of doing it like that?

Reply:
but this causes a bug - this param is set to None elsewhere and as such comes here. this causes an exception on split attempt

Reply:
Yep that's why I changed it. Is there a more pythonic way to approach that?